rest/route/host_spawn.go

```diff
@@ -1148,13 +1148,6 @@ func (h *hostExtendExpirationHandler) Parse(ctx context.Context, r *http.Request
 	}
 	h.addHours = time.Duration(addHours) * time.Hour
 
-	if h.addHours <= 0 {
```

Comment:
We should still at least still return a 400 bad request if the user sends in an invalid expiration extension. Since `MakeExtendedSpawnHostExpiration` in `Run` below is what now handles this, how about we change L1165 to return a 400 code rather than an internal error responder?